application/app/components/fiches/ficheEtablissement/ficheEtablissement.tsx

```diff
@@ -22,11 +22,16 @@ export default function FicheEtablissement({ feature }: FicheEtablissementProps)
 			<br />
 			<ActionButtons />
 			<hr className='my-4' />
-			<ProtectionCard isProtected={!!feature.protection} />
-			<PotentielSolaireCard potentiel_solaire={feature.potentiel_solaire} />
+			{feature.protection && <ProtectionCard />}
+			<PotentielSolaireCard
+				potentiel_solaire={feature.potentiel_solaire}
+				showInterpretation
+				showNbEtablissements={false}
+				nb_eleves={feature.nb_eleves ?? undefined}
```

Comment:
Les props sont en camelCase

```suggestion
			<PotentielSolaireCard
				potentielSolaire={feature.potentiel_solaire}
				showInterpretation
				showNbEtablissements={false}
				nbEleves={feature.nb_eleves ?? undefined}
```